packages/agoric-cli/src/bin-agops.js

```diff
@@ -30,22 +31,22 @@ program.addCommand(await makeOracleCommand(logger));
 program.addCommand(await makeEconomicCommiteeCommand(logger));
 program.addCommand(await makePerfCommand(logger));
 program.addCommand(await makePsmCommand(logger));
-program.addCommand(await makeReserveCommand(logger));
 program.addCommand(await makeVaultsCommand(logger));
 
-program.addCommand(
-  await makeInterCommand(
-    {
-      env: { ...process.env },
-      stdout: process.stdout,
-      stderr: process.stderr,
-      createCommand,
-      execFileSync,
-      now: () => Date.now(),
-    },
-    { fetch },
-  ),
-);
+const procIO = {
+  env: { ...process.env },
+  stdout: process.stdout,
+  stderr: process.stderr,
+  createCommand,
+  execFileSync,
+  now: () => Date.now(),
+  clock: () => Promise.resolve(Date.now()),
+  setTimeout,
+};
+
+program.addCommand(await makeReserveCommand(logger, procIO));
+program.addCommand(makeAuctionCommand(logger, { ...procIO, fetch }));
+program.addCommand(await makeInterCommand(procIO, { fetch }));
```

Comment:
now that you defer lookup these needn't be async,
```suggestion
program.addCommand(makeReserveCommand(logger, procIO));
program.addCommand(makeAuctionCommand(logger, { ...procIO, fetch }));
program.addCommand(makeInterCommand(procIO, { fetch }));
```